test/functionalTest/cases/1367_time_measures/time_measure_stats.test

```diff
@@ -287,26 +284,6 @@ echo
 
 
 --REGEXPECT--
-01. Initial call to GET /statistics (gives NO time stats as LAST /statistics is excluded)
-==================================================================
-HTTP/1.1 200 OK
-Content-Length: REGEX((258|260))
-Content-Type: application/json
-Date: REGEX(.*)
-
-{
-    "measuring_interval_in_secs": "REGEX(\d+)",
-    "statisticsRequests": "1",
-    "subCacheInserts": "0",
-    "subCacheItems": "0",
-    "subCacheRefreshs": "1",
-    "subCacheRemoves": "0",
-    "subCacheUpdates": "0",
-    "uptime_in_secs": "REGEX(\d+)",
-    "versionRequests": "0"
-}
-
-
 02. Second call to GET /statistics (gives time stat of FIRST request for /statistics)
```

Comment:
It is a bit werid a "REGEXPECT" section starting by step 2: user may think that something is wrong and some step is missing. Maybe a solution is to do `echo "01. Pre-call to GET /statistics to give VALGRIND a possibility"` even although the actual step is not shown due it uses `/dev/null` output redirection.